bisect_ppx-ocamlbuild.opam

```diff
@@ -0,0 +1,24 @@
+name: "bisect_ppx-ocamlbuild"
```

Comment:
Having `dev` as a version in a pinned opam file has strange effects like causing upper bounds to break. I'm not sure if the default `~unknown` has the same effect.